backend/src/main/resources/application.yaml

```diff
@@ -21,3 +21,9 @@ spring:
 
 server:
   port: 8080
+---
```

Comment:
dashed lines define profiles. please move it above first dashed lines.

Reply:
Done.